app/src/main/java/ar/com/wolox/android/example/model/youtube/Default.kt

```diff
@@ -0,0 +1,3 @@
+package ar.com.wolox.android.example.model.youtube
+
+data class Default(val url: String, val width: Int, val height: String)
```

Comment:
The same comment will be for Default, high, Id, Medium, etc. Can we rename them with more descriptive names?